### ISSUE TITLE
Service name does not indicate language

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#584]
Issue group: Misc
Visibility: 2
Evaluation: Does not support
Success Criteria: 3.1.2: Language of Parts
Needs design: Interaction, Content

### Description
When creating a service, you are instructed to write the name in both English and French. However, there is no way to indicate which is which and the lang attribute is not set for the service name when used across the site. Rather than provide a single field for the service name, a field for each language should be provided so that when used on the site it could be added with the correct lang code. For example the service Renew your passport | Renouveler votre passeport could be marked up like 
`<h2><span lang=“en-CA”>Renew your passport</span>|<span lang=“fr-CA”>Renouveler votre passeport</span></h2>`

The above is also true for the example service name presented in the instructions.

Chris Coulson (Accessibility Standards Canada) noted for us that:
>it would be helpful from an accessibility and official languages perspective to be able to include multiple images/header images in a template.

**Comment by amazingphilippe** [comment]
We can fix the example, but not the user generated content. Here's a Pr for the first part: https://github.com/cds-snc/notification-admin/pull/1736
